nixos/modules/services/web-apps/whitebophir.nix

```diff
@@ -0,0 +1,54 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.whitebophir;
+in {
+  options = {
+    services.whitebophir = {
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          If enabled, start the online collaborative whiteboard server
+          whitebophir. Persistent state will be maintained under
+          <filename>/var/lib/whitebophir</filename>.
+        '';
+      };
+
+      package = mkOption {
```

Comment:
Is there any variation in what a user might provide here? Maybe this option isn't needed.

Reply:
I got the impression that this is the way it's usually done (for instance cryptpad is the same). I imagine this can be useful if you want to override the service with a custom version of whitebophir, which might actually hapen because whitebophir is so easily hackable. :-)

Reply:
If there are multiple package versions available, build time configuration options, or it is common to specify a custom package this is convenient, otherwise an `overlay` is generally the preferred way to do this in my opinion.

Reply:
I just thought about this again and wanted to remove this option, for the sake of simplicity. But it turns out that this option is important after all, especially in the current state where this pull request is not merged: While it is easy to import the whitebophir service from the local file syste, the default option `pkgs.whitebophir` won't work as long as this pull request is not merged.

Reply:
Overlays still take care of this, but please feel free to keep the option if you find it valuable. As you mentioned it is a fairly common option :+1: